### PR TITLE
Add CLI examples for AWS Cloud Map (servicediscovery)

### PR DESCRIPTION
**Pull request by HLeeHart**
*Add CLI examples for common AWS Cloud Map commands*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by JennieHubbard** [comment]
@kyleknap Is there a reason why this PR has not been merged?

**Comment by atrakic** [comment]
Awesome work. Thx @HLeeHart.

File: doc/source/conf.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 # General information about the project.
 project = u'AWS CLI Command Reference'
-copyright = u'2018, Amazon Web Services'
```
Comment (stealthycoin) [inline doc/source/conf.py:46]: Lets remove this update.

**Review by stealthycoin — COMMENTED** [review]
Just one comment, otherwise it looks good.

File: doc/source/conf.py (hunk 1 above)
Comment (HLeeHart) [reply doc/source/conf.py]: Undone with https://github.com/aws/aws-cli/pull/5141/commits/a02823f04ed982e5ddc3d8111346163085eecb04

**Review by stealthycoin — APPROVED** [review]

**Pull request merged**